File: submissions/kunmi02/week1solution/fizzbuzz.java
public class Fizzbuzz {    
    public static void main(String[] args) {
        // TODO code application logic here
        int endcount=100; //end of my counter
        for (int i=1;i<=endcount;i++){ //initialized a for loop to count from 1 - 100
            
            if ((i % 3)==0 && (i % 5)==0){ // condition for fizzbuzz
                System.out.println ("FizzBuzz" );
            } 
            else if ((i % 3)==0){ //condition fizz
                System.out.println ("Fizz" );
            } 
            else if ((i % 5)==0){ // condition for buzz
                System.out.println ("Buzz" );
            } 
            else{
            System.out.println ( i ); //print the number if it does not satisfy the fizz,buzz or fizzbuzz condition
            }
        }
    }
    
}

File: submissions/j-enny/week1/fizzbuzz.cpp
#include <iostream>
using namespace std;
int main( )
{
	int i = 1;
	int n = 100;
	for (i; i <= n; i++)
	{
		int k = i % 3;
		int j = i % 5;
		if (k == 0 && j == 0)
		{
			cout << "FIZZBUZZ" << endl;
		}
		
		else if (k == 0)
		{
			cout << "FIZZ" << endl;
		}
		else if (j == 0)
		{
			cout << "BUZZ" << endl;
		}
		else
		{
			cout << i << endl;
		}
	}
	return 0;
}
